crates/sui-framework/packages/deepbook/sources/clob.move

```diff
@@ -683,10 +674,9 @@ module deepbook::clob {
                     let filled_base_quantity =
                         if (taker_base_quantity_remaining >= maker_base_quantity) { maker_base_quantity }
                         else { taker_base_quantity_remaining };
-                    // filled_quote_quantity from maker, need to round up, but do in decrease stage
```

Comment:
Was this comment out of date?

Reply:
Yeah,  This commet has been outdated due to the change in the handling fee design.

Reply:
@sblackshear For clarification, the commission fee logic was updated in a PR updated a few weeks ago.